micro_sam/sam_annotator/_widgets.py

```diff
@@ -171,6 +172,35 @@ class PBarSignals(QObject):
     pbar_reset = Signal()
 
 
+class InfoDialog(QDialog):
+    def __init__(self, title, message):
+        super().__init__()
+        self.setWindowTitle(title)
+        self.user_cancelled = False  # Initialize flag
+
+        layout = QVBoxLayout()
+        layout.addWidget(QLabel(message))
+
+        # Add buttons
+        button_box = QHBoxLayout()  # Use QHBoxLayout for buttons side-by-side
+        accept_button = QPushButton("OK") 
+        accept_button.clicked.connect(lambda: self.button_clicked(accept_button))  # Connect to clicked signal
+        button_box.addWidget(accept_button)
+
+        cancel_button = QPushButton("Cancel")
+        cancel_button.clicked.connect(lambda: self.button_clicked(cancel_button))  # Connect to clicked signal
+        button_box.addWidget(cancel_button)
+
+        layout.addLayout(button_box)
+        self.setLayout(layout)
+
+    def button_clicked(self, button):
+        if button.text() == "OK":
+            self.accept()  # Accept the dialog
```

Comment:
Where are `self.accept` and `self.reject` defined? Is this already implemented in `QDialog`?

Reply:
yes, it is already implemented in QDialog - https://doc.qt.io/qt-6/qdialog.html#accept